scripts/bump.sh

```diff
@@ -109,49 +120,168 @@ check_iscsc_remote () {
 	if [ -z "$remote" ]; then
 		log_error "'iScsc/iscsc.fr' remote is not in remote list"
 		exit 1
+	elif [ "$remote" == "${HTTPS_REMOTE}" ]; then
+		log_warning "Using HTTPS remote, SSH remote not found"
 	fi
 	log_ok "'iScsc/iscsc.fr' is in remote list as '$remote' OK"
```

Comment:
Since I see that you changed iScsc to a lower case (as per Atxr's request), maybe the log should be consistent with this change

Reply:
I didn't 😉 I just added `--ignore-case` toi greps, anyway in this log I refer to the protocol, thus the upper case!